styles.css

```diff
@@ -0,0 +1,391 @@
+*{
+	box-sizing: border-box;
+}
+
+body {
+    max-width: 80rem;
+    margin: auto;
+}
+
+
+header {
+	height:10%;
```

Comment:
make sure you have spaces after the colon. Remember, the formatting should look like...

h1 {
  color: #f2f2f2;
}